### PR TITLE
quicktour.mdx en -> pt translation

### PR DESCRIPTION
**Pull request by vitorfrois**
# What does this PR do?

Translate en/quicktour.mdx into a brazilian portuguese version.
May be worth checking some words and terms translation to maintain continuity between all portuguese translations.

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [X] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [x] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by omarespejel** [comment]
Thank you @vitorfrois for the PR! Sorry for the late reply I lost track of the PR. Relates to  #16824.

@sgugger LGTM :). I think the test error in "`Build PR Documentation / build / build_pr_documentation`" relates mostly to a JS issue.

**Review by omarespejel — APPROVED** [review]

File: docs/source/pt/quicktour.mdx [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+...     padding=True,
+...     truncation=True,
+...     max_length=512,
+...     return_tensors="pt",
+... )
+```
+
+Leia o tutorial de [pré-processamento](./pré-processamento) para obter mais detalhes sobre tokenização.
+
+### AutoModel
```
Comment (sgugger) [inline docs/source/pt/quicktour.mdx:239]: This `<frameworkcontent>` is opened but not closed, which is the reason the Build PR job fails.  Could you fix this by making sure to do the same as in the English version?

**Review by sgugger — COMMENTED** [review]
Thanks for this new translation. There is a syntax problem in the new MDX file, which is why the build job fails.

File: docs/source/pt/quicktour.mdx (hunk 1 above)
Comment (vitorfrois) [reply docs/source/pt/quicktour.mdx]: Fixed it in portuguese. The english version is working well

**Comment by sgugger** [comment]
Thanks a lot!

**Pull request merged**